model/Hardware/Classes/UseAction.md

```diff
@@ -0,0 +1,26 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# UseAction
+
+## Summary
+
+The action of product modification.
+
+## Description
+
+This is the specific action of product modification.
+
+Relationship: 
+For Each `UseAction` there is at least one `/Core/Relationship` class or subclass with the relationshipType of 'performedBy’ on the from and an `/Core/Agent` class or subclass on the to. 
+For Each `UseAction` there is at least one `/Core/Relationship` class or subclass with the relationshipType of 'effected’ on the from and an `/Core/Element` class or subclass on the to. 
+
+## Metadata
+
+- name: UseAction
+- SubclassOf: /Core/Action
+- Instantiability: Abstract
```

Comment:
```suggestion
- Instantiability: Concrete
```

I'm not sure about this, but does `UseAction` intended to be a concrete class of `ModifyAction`?
If not, please ignore this suggestion.